schemas/twitter.js

```diff
@@ -0,0 +1,167 @@
+        screen_name: { type: GraphQLString },
+        name: { type: GraphQLString },
+        profile_image_url: { type: GraphQLString },
+        url: { type: GraphQLString },
+        tweets_count: { 
+            type: GraphQLInt,
+            resolve: ({ statuses_count }) => statuses_count
+        },
+        followers_count: { type: GraphQLInt },
+        tweets: {
+            type: new GraphQLList(TweetType),
+            description: 'Get a list of tweets for current user',
+            args: {
+                limit: { type: GraphQLInt }
+            },
+
+            //             user            args
+            resolve: ({ id: user_id }, { limit = 10 }) => twitter.getTweets(user_id, limit)
+        }
+    })
+
+});
+
+let TweetType = new GraphQLObjectType({
+
+    name: 'Tweet',
+    description: 'A tweet object',
+    fields: () => ({
+        id: { type: GraphQLID },
+        created_at: { type: GraphQLString },
+        text: { type: GraphQLString },
+        retweet_count: { type: GraphQLInt },
+        user: {
+            type: UserType
+        },
+        retweets: {
+            type: new GraphQLList(RetweetType),
+            description: 'Get a list of retweets',
+            args: {
+                limit: { type: GraphQLInt }
+            },
+            //        passing integer 'id' here doesn't work surprisingly, had to use 'id_str'
+            resolve: ({ id_str: tweetId }, { limit = 5 }) => twitter.getRetweets(tweetId, limit)
+        }
+    })
+});
+
+let RetweetType = new GraphQLObjectType({
+
+    name: 'Retweet',
+    description: 'Retweet of a tweet',
+    fields: () => ({
+        id: { type: GraphQLID },
+        created_at: { type: GraphQLString },
+        in_reply_to_tweet_id: {
+            type: GraphQLString,
+            resolve: ({ in_reply_to_status_id }) => in_reply_to_status_id
+        },
+        in_reply_to_user_id: { type: GraphQLInt },
+        in_reply_to_screen_name: { type: GraphQLString },
+        retweeted_status: {
+            type: TweetType
+        },
+        user: {
+            type: UserType
+        }
+    })
+});
+
+let twitterType = new GraphQLObjectType({
+
+    name: 'TwitterAPI',
+    description: 'The Twitter API',
+    fields: {
+        user: {
+            type: UserType,
+            args: {
+                user_id: {
+                    description: 'ID of user account',
+                    type: GraphQLInt
+                },
+                screen_name: {
+                    description: 'Screenname of the user',
+                    type: GraphQLString
+                }
+            },
+            resolve: (_, { user_id, screen_name }) => {
+
+                const { getUser } = twitter;
+
+                if (!user_id && !screen_name) {
+                    return getUser('user_id', 9533042);
+                }
+                else {
+
+                    if (isNaN(parseInt(user_id))) {
```

Comment:
I can add the entire lodash package if you want. (knowing that _.isNumber() simply equals doing `Object.prototype.toString.call(value) ===   '[object Number]'` )